java/src/main/java/lesson03/README.md

```diff
@@ -122,14 +121,6 @@ and we mark the span with a `span.kind=client` tag, as recommended by the OpenTr
 
 Our servers are currently not instrumented for tracing. We need to do the following:
 
-#### Add some imports
```

Comment:
Why do you think this should be removed? Without looking at the solution it's hard for a new user to know what to import (e.g. they might mistakenly import Jaeger classes instead of opentracing)

Reply:
Sounds good @jpkrohling.